src/test/unit/dronecan_gnss_unittest.cc

```
@@ -0,0 +1,429 @@
+    memset(&t, 0, sizeof(t));
+    t.payload_head = payload;
+    t.payload_len = len;
+    t.payload_middle = NULL;
+    t.payload_tail = NULL;
+    handler(NULL, &t);
+}
+
+static void feedDefaultFix2(void)
+{
+    uint8_t buf[64];
+    feed(handleFix2, buf, buildFix2(buf));
```

Cppcheck (2.21.0)

[error] Uninitialized variable

(uninitvar)